docker-bake.hcl

```diff
@@ -51,6 +52,14 @@ variable "JAVA17_VERSION" {
   default = "17.0.7_7"
 }
 
+variable "DEBIAN_RELEASE" {
+  default = "bullseye-20230522"
+}
+
+variable "DEBIAN_NEXT_RELEASE" {
+  default = "bookworm-20230522"
```

Comment:
Same comment as for the Dependabot update: we have to track this version

Reply:
No problem, I can help if you want to split the burden?

Reply:
I don't mind. 
I would rather see this merged faster than slower :) .

Reply:
I think only the automatic version update is left to do.

Reply:
Let's solve this in the new PR.